x-pack/plugins/actions/server/builtin_action_types/es_index.ts

```diff
@@ -92,8 +75,8 @@ async function executor(
     body: bulkBody,
   };
 
-  if (params.refresh != null) {
-    bulkParams.refresh = params.refresh;
+  if (config.refresh != null) {
+    bulkParams.refresh = config.refresh;
```

Comment:
nit: I believe `config.refresh` will now always have a value with the updated validation. We should be good to define it directly in `bulkParams` now (without check for null).